src/main/java/seedu/address/model/score/Score.java

```diff
@@ -0,0 +1,109 @@
+package seedu.address.model.score;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+import seedu.address.model.score.exceptions.BadDateException;
+
+/**
+ * Represents a Score in the address book.
+ * Guarantees: immutable; fields are validated; details are present and not null;
+ */
+public class Score {
+    // Identity field(s)
+    public final Title scoreName;
```

Comment:
Do we really need the title class itself? I feel if we do this way, then we may follow the AB3 person format as socreValue and scoreDate should have individual classes also.

Reply:
Main reason is that if there's a different format of a regex we are following, we can apply the validation over.